### PR TITLE
Setup cypress test for book

### LINKED ISSUE #1548: vue-book test
The idea is to have an automated test, that would go through vue-book and check there are no errors on any of the pages.

Here's implementation from different project, we might be able to reuse it:

```ts
const browser = await create_spec_browser();
const context = await browser.newContext();
const page = await context.newPage();
page.setViewportSize(default_viewport);
await page.goto(book_endpoint)

let errorCount = 0
let currentDemo = ''
page.on('console', msg => {
  if (msg.type() === 'error') {
    console.error(`${currentDemo}: ${msg.text()}`)
    errorCount++
  }
})

const files = await page.$$('.BookComponentListItem')
for (const index in files ) {
  await files[index].click()
  currentDemo = await files[index].textContent()
  await sleep(500)
}
expect(errorCount).toBe(0)
await sleep(5000)
```

### PR DESCRIPTION
**Pull request by LighthouseKeeperYN**
close #1548

No automation for now, to run test execute "yarn test:e2e" and click on the test you want to run in appeared window.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/cypress/examples/1-getting-started/todo.spec.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+// please read our getting started guide:
+// https://on.cypress.io/introduction-to-cypress
+
+describe('example to-do app', () => {
+  beforeEach(() => {
+    // Cypress starts out with a blank slate for each test
+    // so we must tell it to visit our website with the `cy.visit()` command.
+    // Since we want to visit the same URL at the start of all our tests,
+    // we include it in our beforeEach function so that it runs before each test
+    cy.visit('https://example.cypress.io/todo')
```
Comment (m0ksem) [inline packages/ui/cypress/examples/1-getting-started/todo.spec.js:20]: Is this PR still in progress?

Comment (LighthouseKeeperYN) [reply packages/ui/cypress/examples/1-getting-started/todo.spec.js:20]: those are just an examples. I deiced to leave them for quick reference.

**Comment by asvae** [comment]
I suggest to remove examples from this repo. No much value.

**Comment by damirJa** [comment]
Guys I've fixed tests script, added fixtures generator and script for CI usage, please check it out

**Comment by m0ksem** [comment]
Sync with develop, but `yarn test:e2e` not working for me.

```bash
ERROR  TypeError: runner.on is not a function
TypeError: runner.on is not a function
```

**Comment by asvae** [comment]
Closing this because we have plans to use storybook, which handles all our cases out of the box

**Pull request closed without merging**